### PR TITLE
[8.11.x] PLANNER-2474 optaplanner CI (#141)

### PR DESCRIPTION
**Pull request by radtriste**
# JIRA
https://issues.redhat.com/browse/PLANNER-2474

# Cherry-pick
https://github.com/kiegroup/optaplanner-quickstarts/pull/141

# Related PRs:
* https://github.com/kiegroup/kogito-pipelines/pull/246
* https://github.com/kiegroup/optaplanner/pull/1529
* https://github.com/kiegroup/optaplanner-quickstarts/pull/165
* https://github.com/kiegroup/optaweb-employee-rostering/pull/681
* https://github.com/kiegroup/optaweb-vehicle-routing/pull/551

### CODE REVIEW

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

File: .github/workflows/full-downstream.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      - 'LICENSE*'
+      - '.gitignore'
+      - '**.md'
+      - '**.adoc'
+      - '*.txt'
+      - '.ci/**'
+
+jobs:
+  concurrency:
+    group: pull_request-${{ github.head_ref }}
```
Comment (rsynek) [inline .github/workflows/full-downstream.yml]: Should we use the same group as we do in the pull-request.yml?
If this needs to change, we have to backport to the `development` branch.

**Review by rsynek — APPROVED** [review]
LGTM, one minor suggestion.

File: .github/workflows/full-downstream.yml (hunk 1 above)
Comment (radtriste) [reply .github/workflows/full-downstream.yml]: @Ginxo any opinion on this ?

Comment (radtriste) [reply .github/workflows/full-downstream.yml]: cherry-picking should solve that

**Comment by radtriste** [comment]
jenkins retest this

**Pull request merged**